structs.go

```diff
@@ -2174,7 +2174,7 @@ func (activity *Activity) UnmarshalJSON(b []byte) error {
 		Type          ActivityType `json:"type"`
 		URL           string       `json:"url,omitempty"`
 		CreatedAt     int64        `json:"created_at"`
-		ApplicationID string       `json:"application_id,omitempty"`
+		ApplicationID interface{}  `json:"application_id,omitempty"`
```

Comment:
If it's only an integer number, I think it'd be better to use `json.Number` for this

Reply:
Good shout - I've just made that change